sriov/main.go

```diff
@@ -32,8 +34,11 @@ func cmdAdd(args *skel.CmdArgs) error {
 	}
 	defer netns.Close()
 
+	oldIfname := os.Getenv("CNI_IFNAME")
+	defer os.Setenv("CNI_IFNAME", oldIfname)
 	if n.IF0NAME != "" {
 		args.IfName = n.IF0NAME
+		os.Setenv("CNI_IFNAME", args.IfName)
 	}
```

Comment:
The IF0NAME field was very old feature introduced a long time before any existence CRD based metaplugin. This was a way of bypassing CNI_IFNAME from env. This is no longer needed and we plan to remove it from the sriov NetConf struct. 
This plugin should work with the CNI_IFNAME set by user/metaplugin.

May I ask why do you need to set the  IF0NAME?